projects/go-lib/src/lib/components/go-datepicker/go-calendar.component.ts

```diff
@@ -121,5 +122,25 @@ export class GoCalendarComponent implements OnDestroy, OnInit {
     }
     this.currentMonth = this.selectedDate.getMonth();
     this.updateYear(this.selectedDate.getFullYear());
+    if (this.appendTo === 'body') {
+      setTimeout(() => {
+        const calenderBody: any = document.getElementsByClassName(
+          'append-body'
+        )[0];
```

Comment:
This will break if we have an instance with multiple datepickers on a page. What if we changed this to an Angular `ViewChild` reference instead?

Reply:
Yes, you are right. I have removed and update the code here.